tests/testthat/test-add_overall.R

```diff
@@ -7,17 +7,32 @@ test_that("multiplication works", {
         ae = adverse_event,
```

Comment:
This comment isn't in the right place, sorry.  Wasn't letting me review line 1.

```suggestion
        test_that("add_overall() works", {
```

Reply:
just pushed the updated comment!